float32 power
---
int8 status